chroma_core/models/ldev_configuration.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020 DDN. All rights reserved.
+# Use of this source code is governed by a MIT-style
+# license that can be found in the LICENSE file.
+
+from chroma_core.lib.job import Step
+from chroma_core.lib.util import invoke_rust_agent
+from chroma_core.models import Job
+from chroma_help.help import help_text
+from django.contrib.postgres import fields
+
+import json
+
+
+class ConfigureLDevStep(Step):
+    def run(self, kwargs):
+        ldev_entries = kwargs["ldev_entries"]
+        ldev_entries = json.loads(ldev_entries)
+
+        for (fqdn, entries) in ldev_entries.items():
+            self.invoke_rust_agent(fqdn, "create_ldev_conf", entries)
```

Comment:
Why not: `self.invoke_rust_agent_expect_result`?

Reply:
Updated